Gems/Atom/RPI/Code/Include/Atom/RPI.Public/ViewportContext.h

```diff
@@ -61,11 +61,21 @@ namespace AZ
             //! renamed via AZ::RPI::ViewportContextRequests::Get()->RenameViewportContext(...).
             AZ::Name GetName() const;
 
+            //! Gets the view group associated with this ViewportContext.
+            //! Alternatively, use AZ::RPI::ViewportContextRequests::Get()->GetCurrentViewGroup().
+            ViewGroupPtr GetViewGroup();
+            ConstViewGroupPtr GetViewGroup() const;
+
             //! Gets the default view associated with this ViewportContext.
-            //! Alternatively, use  AZ::RPI::ViewportContextRequests::Get()->GetCurrentView().
+            //! Alternatively, use AZ::RPI::ViewportContextRequests::Get()->GetCurrentViewGroup()->GetView().
             ViewPtr GetDefaultView();
             ConstViewPtr GetDefaultView() const;
 
+            //! Gets the stereoscopic view associated with this ViewportContext.
+            //! Alternatively, use AZ::RPI::ViewportContextRequests::Get()->GetCurrentViewGroup()->GetView(AZ::RPI::ViewType::XrLeft|XrRight).
```

Comment:
The comment seems to imply you can pass AZ::RPI::ViewType::XrLeft|XrRight into this function which I dont think is correct. Maybe just remove XrLeft|XrRight part of the comment.

Reply:
done